bootstrap.sh

```diff
@@ -1 +1,10 @@
 #!/bin/bash
+
+kind create cluster --config cluster.yml
+kubectl apply -f mysql-configMap.yml
+kubectl apply -f mysql-secret.yaml
```

Comment:
Ensure that the file name `mysql-secret.yaml` matches the actual file name. It should be `mysql-secret.yml` if following the naming convention from the other files.